client/components/tinymce/plugins/simple-payments/dialog/form.jsx

```diff
@@ -19,121 +23,128 @@ import CompactFormToggle from 'components/forms/form-toggle/compact';
 import FormInputValidation from 'components/forms/form-input-validation';
 import UploadImage from 'blocks/upload-image';
 
-class ProductForm extends Component {
-	handleFieldChange = ( { currentTarget: { name, value } } ) => {
-		this.props.onFieldChange( name, value );
-	};
+const REDUX_FORM_NAME = 'simplePaymentsForm';
+
+// Export some selectors that are needed by the code that submits the form
+export const getProductFormValues = state => getFormValues( REDUX_FORM_NAME )( state );
+export const isProductFormValid = state => isValid( REDUX_FORM_NAME )( state );
+export const isProductFormDirty = state => isDirty( REDUX_FORM_NAME )( state );
+
+// Validation function for the form
+const validate = ( values, props ) => {
+	// The translate function was passed as a prop to the `reduxForm()` wrapped component
+	const { translate } = props;
+	const errors = {};
+
+	if ( ! values.title ) {
+		errors.title = translate( 'Product name can not be empty.' );
+	}
+
+	if ( ! values.price ) {
+		errors.price = translate( 'Price can not be empty.' );
```

Comment:
We should probably use "cannot" instead of "can not" since it's more formal and @a8ci18n bot is suggesting that in the above comment. In all places where we use "can not".